### ISSUE TITLE
[FEATURE] Request Patch "Openning.bik"

### ISSUE THREAD

**Issue by RagnaWay** [Neo-Mind/WARP#9]
In new 2020-xx-xx clients **openning.bik** is already active, but you need a patch to activate this function.

![image](https://user-images.githubusercontent.com/53480357/106503148-09fe3180-64a4-11eb-9a9d-6b6d39fb5797.png)

**Comment by Everade** [comment]
**Additional Info:**
RagnarokZero clients are the only clients i know that actualy play the Openning.bik upon starting the client.
Also the "Opening" button on the login screen will play the Openning.bik video when pressed.

So it should most likely be possible to take a look at the RagnarokZero client to be able to activate this feature on other clients.

Talking about this button by the way:
https://github.com/Neo-Mind/WARP/issues/35

**Comment by Neo-Mind** [comment]
hmm, so what is the expectation with this patch?
Play the bik file when the openning button is clicked ?

**Comment by Everade** [comment]
From my perspective, yes.
![image](https://user-images.githubusercontent.com/12261601/121924654-67509c00-cd3c-11eb-8643-481a381bd50a.png)

**Comment by RagnaWay** [comment]
> hmm, so what is the expectation with this patch?
> Play the bik file when the openning button is clicked ?

In the official openning.bik plays the video every time you open the game, and you can also see the video again through the button on the login screen.

**Comment by Everade** [comment]
Yes the Zero client does that, however it's pretty annoying if you're playing frequently.
Would personally prefer to only enable the button.

And maybe add a secondary setting for the auto-play for those who're interested into it.
